### DISCUSSION TITLE
Get my Vercel project custom domain name to appear in address bar

### DISCUSSION THREAD

**Discussion by estellarspace** [vercel/community#3565] [Help] [answered]
### Summary

I added "estellarspace.com" in Domains and I believe it's configured properly.

When I go to estellarspace.com or www.estellarspace.com it redirects to the project, and the project URL appears in the address bar, "https://nextjs-alpha-kohl-81.vercel.app/".

This is progress. However, the company really wants to 'only' show estellarspace.com in the address bar.
This is probably something silly. What am I missing, please?

### Steps to Reproduce

Go to estellarspace.com
See the address bar showing a different URL (the Vercel one)

**Comment by amyegan** [comment, marked as answer]
Hey @estellarspace. It looks like you have `estellarspace.com` set up to redirect to `nextjs-alpha-kohl-81.vercel.app`. If you want the custom domain to show in the address bar, you need to change the settings for that domain so `Redirect to` is set to `No Redirect` and the `Git Branch` field is left blank.

**Comment by estellarspace** [comment]
Oh, OK, works. Thank you @amyegan !
